porch/controllers/functiondiscovery/testdata/testfunctiondiscoverycontroller/oci/expected.yaml

```diff
@@ -0,0 +1,163 @@
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: Function
+metadata:
+  creationTimestamp: null
+  name: kpt-functions.enable-gcp-services.v0
+  namespace: default
+spec:
+  description: description for enable-gcp-services:v0
+  documentationUrl: documentation for enable-gcp-services:v0
+  image: ociurl/enable-gcp-services:v0
+  repositoryRef:
+    name: kpt-functions
+status: {}
+
+---
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: Function
+metadata:
+  creationTimestamp: null
+  name: kpt-functions.enable-gcp-services.v0.0
+  namespace: default
+spec:
+  description: description for enable-gcp-services:v0.0
+  documentationUrl: documentation for enable-gcp-services:v0.0
+  image: ociurl/enable-gcp-services:v0.0
+  repositoryRef:
+    name: kpt-functions
+status: {}
+
+---
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: Function
+metadata:
+  creationTimestamp: null
+  name: kpt-functions.enable-gcp-services.v0.0.0
+  namespace: default
+spec:
+  description: description for enable-gcp-services:v0.0.0
+  documentationUrl: documentation for enable-gcp-services:v0.0.0
+  image: ociurl/enable-gcp-services:v0.0.0
+  repositoryRef:
+    name: kpt-functions
+status: {}
+
+---
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: Function
+metadata:
+  creationTimestamp: null
+  name: kpt-functions.set-labels.unstable
+  namespace: default
+spec:
+  description: description for set-labels:unstable
+  documentationUrl: documentation for set-labels:unstable
+  image: ociurl/set-labels:unstable
+  repositoryRef:
+    name: kpt-functions
+status: {}
+
+---
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: Function
+metadata:
+  creationTimestamp: null
+  name: kpt-functions.set-labels.v0
```

Comment:
Ignore my previous question, I can see how versions are part of the name.

May be we will have to add label for `function-name` so that one can retrieve all the versions of a function using labelSelector ?

Reply:
I think we will yes, but I'd like to tackle that as part of thinking about what we want to expose here.  I can see us exposing a list of versions in the status as well, for example, rather than one object per version.